README.md

```diff
@@ -84,7 +84,7 @@ const headers = req.headers;
 
 const isValid = payU.verifyNotification(
   headers["OpenPayu-Signature"],
-  JSON.stringify(req.body)
+  req.body
```

Comment:
i think in express we gotta use `req.rawBody` with `row-body-parser`

Reply:
we gotta add a comment about this

Reply:
sure, I added some note about what need to be passed into rawBody, IMHO information that is framework specific is more universal than giving specific examples...